tests/versioning.go

```diff
@@ -3416,6 +3416,25 @@ func (s *VersioningIntegSuite) TestDescribeTaskQueueEnhanced_TooManyBuildIds() {
 	s.Nil(resp)
 }
 
+func (s *VersioningIntegSuite) TestDescribeTaskQueue_IsRateLimited() {
```

Comment:
This is overkill IMHO and could also fail if you request frequently enough for a "normal" (non-visibility API).